app/src/main/java/com/example/nasaapp/data/value_object/Data.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.nasaapp.data.value_object
+
+
+import com.google.gson.annotations.SerializedName
+
+data class Data(
+    val album: List<String>,
```

Comment:
Для всех Dto (Data transfer objects) нужно указывать аннотацию для парсинга
```suggestion
    @SerializedName("album")
    val album: List<String>,
```

Reply:
также замени все data классы на обычный class. И поставь todo добавить конвертацию в Domain models